js/main.js

```diff
@@ -91,9 +91,9 @@ function loadEvents(days){
   var ce = $('.card-container__events');
   for(var i = 0; i < days.length; i++) {
     var day = days[i];
-    var ds = day.day+'-event';
-    var ul = day.day+'-events';
-    var de = $('<li><div id="'+day.day+'" class="collapsible-header container day"><h5 class="day__title" data-position="right"><span class="day__title__bold">' + day.day + '</span> - ' + day.description + '</h5></div><div class="collapsible-body"><ul id="'+ul+'"class="collapsible sub-collapsible" data-collepsible="accordion"></div></ul>').appendTo(ce);
+    var ul = day.day + '-events';
+    var de = $('<li><div id="' + day.day + '" class="collapsible-header container day"><h5 class="day__title" data-position="right"><span class="day__title__bold">' + day.day + '</span> - ' + day.description + '</h5></div><div class="collapsible-body"><ul id="'+ul+'"class="collapsible sub-collapsible" data-collepsible="accordion"></div></ul>');
```

Comment:
I realise this line was long before, but maybe split it to make it less terrible?